crates/runtime/src/dataconnector/listing/connector.rs

```diff
@@ -135,10 +137,23 @@ pub trait ListingTableConnector: DataConnector {
                 Some(self.get_csv_format(params)?),
                 extension.unwrap_or(".csv".to_string()),
             )),
-            Some("parquet") => Ok((
-                Some(Arc::new(ParquetFormat::default())),
-                extension.unwrap_or(".parquet".to_string()),
-            )),
+            Some("parquet") => {
+                let mut table_parquet_options = TableParquetOptions::new();
+                table_parquet_options
+                    .set("pushdown_filters", "true")
```

Comment:
A simpler option might be to just set the `datafusion.execution.parquet.pushdown_filters ` option on the session context: https://datafusion.apache.org/user-guide/configs.html

Reply:
It wono't work. I've tried that in `DataFusionBuilder`. It's because the datafusion ParquetExec take the filter pushdown configuration from the `ParquetFormat` options instead of session context, so it needs to be set in the `TableParquetOptions ` for `ParquetFormat`: https://github.com/apache/datafusion/blob/444a673682a4823b98695568ecdab1087b5b6a60/datafusion/core/src/datasource/file_format/parquet.rs#L409